### ISSUE TITLE
Can not start proof of access on Appimage

### ISSUE THREAD

**Issue by unklebonehead** [spknetwork/3Speak-app#47]
Using Appimage on Pop!_OS 22.04
3Speak starts and runs just fine. But will not start Proof of access.
All other functions of the app work as normal.

**Comment by vaultec81** [comment]
I dont think we have proofofaccess binaries created for Linux yet. cc: @nathansenn 

We should be able to make those and integrate it into the build process.

**Comment by unklebonehead** [comment]
Thank you!
